src/Symfony/Component/Messenger/Bridge/AmazonSqs/Transport/Connection.php

```diff
@@ -284,12 +290,10 @@ public function send(string $body, array $headers, int $delay = 0, ?string $mess
             'MessageAttributes' => [],
         ];
 
-        foreach ($headers as $name => $value) {
-            $parameters['MessageAttributes'][$name] = new MessageAttributeValue([
-                'DataType' => 'String',
-                'StringValue' => $value,
-            ]);
-        }
+        $parameters['MessageAttributes'][self::MESSAGE_ATTRIBUTE_NAME] = new MessageAttributeValue([
```

Comment:
This will make inter-operability harder with other tools publishing or reading with SQS. Wouldn't it be better to keep setting attributes directly when their name are valid SQS attribute names ?

Reply:
good idea